dvc/repo/pkg/__init__.py

```diff
@@ -6,3 +6,8 @@ def install(self, *args, **kwargs):
         from dvc.repo.pkg.install import install
 
         return install(self.repo, *args, **kwargs)
+
+    def imp(self, *args, **kwargs):
+        from dvc.repo.pkg.imp import imp
+
+        return imp(self.repo, *args, **kwargs)
```

Comment:
Why not just:
```python
return self.repo.imp("pkg://...", ...)
```
?

Reply:
Or just remove this completely and make people use `Repo.imp()` directly?

Reply:
To avoid forcing people to use that prefix and to provide CLI -> API mapping.

Reply:
Doesn't look worth complication to me.

Reply:
@Suor there was a discussion with @dmpetrov and there are a few reasons to provide a higher level command `pkg import` on top of a regular `import` we have already. There is a possibility that in the future they might diverge a little bit, at least in terms of certain "defaults". Also, `pkg import` will be exposing more package specific options. 

For now the major concern was that we need a simple command that can take a Gihub repo and a path inside and pull an artifact from it. With `dvc import` you have to use `pkg://` urls, also you have to provide an alias for the package, otherwise the syntax is getting even more complicated: `pkg+gig@...`. Even if it's fine for some engineers there was a concern that it would be to "geeky" for DS guys. Thus this second "syntax sugar". We can discuss this on the next meeting.

Reply:
@shcheklein I was referring to creating a 2 line file for a function that could be easily inlined. Not talking about cli API. But, ok, let's postpone the discussion.

Reply:
@Suor Ah, sorry, I didn't get that. You are right, it is too much to have a few files for it. I've made the adjustments. Thanks! :slightly_smiling_face: